src/App.tsx

```diff
@@ -1,5 +1,39 @@
+import logo from './assets/images/logo.png'
+import profile_img from './assets/images/profile_img.png'
+import { BiChevronDown } from 'react-icons/bi'
+import { IoChevronBackOutline } from 'react-icons/io5'
+
 function App() {
-  return <h1 className="bg-yellow-500 text-xl">hello world</h1>
+  return (
```

Comment:
please don't create any Tsx inside here. have a different component that handles this. ensure you have a good naming convention.

Reply:
resolved